### ISSUE TITLE
Screen Reader: Settings-Profile: The button 'Account' is announced without its visible name

### ISSUE THREAD

**Issue by nbhardwaj-del** [Expensify/App#75469]
## Action Performed
Prerequisites:
- The user is signed in

1. Using JAWS + Chrome, open the site https://new.expensify.com/settings/profile
2. Press Tab key to focus the button 'Account'.
3. Note how the element is displayed and announced.

## Expected Result
The element's accessible name (i.e., the label that is announced) should contain the visible label of the element. Any additional information added to the label could be appended in front of or behind the visible label. This helps ensure that speech input users can rely on visible labels as a means to interact with the components, as well as gives the same experience to screen reader users.

In this case, the screen reader would announce:
- 'Account, My settings'
- 
## Actual Result
The button is announced without its visible name 'Account', the user can hear only:
- 'My Settings'

HTML:
<button aria-label="My settings" role="button" data-selection-scrapper-hidden-element="true" data-tag="pressable" tabindex="0" class="css-g5y9jx r-1otgn73 r-1loqt21 r-1awozwy r-6koalj r-wy61xf r-1777fci r-n7gxbd r-9ffhgg r-lrvibr" type="button" data-component="GenericPressable" data-element="Pressable" data-source-file="BaseGenericPressable.tsx"><div class="css-g5y9jx r-8jfcpp" data-component="OfflineWithFeedback" data-element="View" data-source-file="OfflineWithFeedback.tsx"><div class="css-g5y9jx"><div class="css-g5y9jx r-bnwqim"><div class="css-g5y9jx r-8drixv r-y47klf r-d045u9 r-zx9znk r-ub6tad r-1qefu2b r-18nhz7w r-5orghg r-u8s1d" data-testid="avatar-ring"></div><div class="css-g5y9jx r-a1yn9n r-h0d30l r-1iww7jx"><div class="css-g5y9jx r-633pao" data-testid="Avatar"><div class="css-g5y9jx"><div class="css-g5y9jx" data-testid="SvgFallbackAvatar Icon" style="overflow: hidden; border-radius: 100px;"><svg xmlns="[http://www.w3.org/2000/svg"](http://www.w3.org/2000/svg%22) xml:space="preserve" id="default-avatar_8_svg__Layer_1" x="0" y="0" viewBox="0 0 80 80" width="28" height="28" hovered="false" pressed="false"><style>.default-avatar_8_svg__st0{fill:#03d47c}.default-avatar_8_svg__st1{fill:#008c59}.default-avatar_8_svg__st2{fill:#002140}.default-avatar_8_svg__st3{fill:#8eecc4}</style>....</svg></div></div></div><div class="css-g5y9jx" style="align-items: center; justify-content: center; border-color: rgb(6, 27, 9); border-radius: 8px; border-width: 2px; position: absolute; right: -4px; top: -3px; height: 12px; width: 12px; z-index: 10; background-color: rgb(242, 87, 48);"></div></div></div></div></div><div dir="auto" class="css-146c3p1" style="color: rgb(231, 236, 233); font-size: 11px; text-align: center; font-family: &quot;Expensify Neue&quot;, &quot;Windows Segoe UI Emoji&quot;, &quot;Noto Color Emoji&quot;, &quot;Custom Emoji Font&quot;; font-style: normal; font-weight: 700; margin-top: 2px; line-height: 14px;">Account</div></button>

## Area issue was found
Settings–Profile

## Failed Checkpoint
WCAG 2.5.3 Label in Name
https://www.w3.org/WAI/WCAG22/Understanding/label-in-name

## User Impact
Speech-input users can directly activate controls on a page with fewer surprising changes of focus. Text-to-speech users will have a better experience because the labels that they hear match the visible text labels that they see on the screen.

## Suggested Resolution
Ensure that the visible text is included in the accessible name by updating the aria-label attribute, e.g:
- 'Account, My settings'
https://www.w3.org/WAI/WCAG22/Techniques/aria/ARIA6

## Environment(s)
Windows Windows 10 - Chrome

## Screenshots/Videos
https://github.com/user-attachments/assets/8e226f91-d727-4aa8-81c4-9f9805a8de5b

<img width="1919" height="999" alt="Image" src="https://github.com/user-attachments/assets/0d4ef9fb-8e1d-40e8-ae6b-64ce0c9e96a4" />

**Version Number:** NA
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** NA
**Logs:** https://stackoverflow.com/c/expensify/questions/4856 NA
**Notes/Photos/Videos:** Any additional supporting documentation Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36605/community-issues/6985970
**Issue reported by:** NA
**Slack conversation:** NA
